runtime/vm/jvminit.c

```diff
@@ -2831,6 +2831,12 @@ checkArgsConsumed(J9JavaVM * vm, J9PortLibrary* portLibrary, J9VMInitArgs* j9vm_
 		findArgInVMArgs( PORTLIB, j9vm_args, EXACT_MATCH, VMOPT_XXDISABLESHAREUNSAFECLASSES, NULL, TRUE);
 	}
 
+	/* consume options for capturing command line in environment variable */
+	if (!ignoreUnrecongizedXXColonOptions) {
```

Comment:
They can always be consumed. Pls put these in consumeVMArgs() with the rest.

Reply:
Moved to `consumeVMArgs` in https://github.com/eclipse-openj9/openj9/commit/05a94161dd11b8fbc081f156ce443b11483e9291